src/lib/components/forms/form/form-components/select/select-box.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-form-field
+    class="w-100 form-input"
+    [tcTooltip]="tooltipText"
+    [placement]="tooltipPlacement"
+    [onlyEllipsisTooltip]="true"
+>
+    <mat-label attr.for="{{ name }}">
```

Comment:
```suggestion
    <mat-label>
```
Das regelt material selbst